packages/core/src/Serialization/Transaction.ts

```diff
@@ -98,8 +99,8 @@ export class Transaction {
    * @param cbor The CBOR encoded transaction object.
    * @returns The new transaction instance.
    */
-  static fromCbor(cbor: HexBlob): Transaction {
-    const reader = new CborReader(cbor);
+  static fromCbor(cbor: TxCBOR): Transaction {
```

Comment:
Isn't this a breaking change?

Reply:
It is not at runtime, but could break builds :sweat_smile: nice catch, reworded the commit